src/test/frontend/handler/paste_test.cljs

```diff
@@ -0,0 +1,32 @@
+(ns frontend.handler.paste-test
```

Comment:
May add more test cases as there are lots of issues with label `copy-paste`

Reply:
Thanks for adding tests here. Agree we need more paste tests as we have too many issues with it